application/views/catalog/partials/advanced_search.php

```diff
@@ -101,6 +101,10 @@
 	         </div>
 	       </div>
 
+               <h5>Still can't find what you're looking for? Try searching <a rel="noopener" target="_blank" href="https://archive.org/details/librivoxaudio">Internet Archive's LibriVox Audiobook Collection</a>
```

Comment:
I tend to dislike messing with the defaults for opening links, as it breaks expectations and removes control from the user (middle click to open in new tab only if I chose to), is there a reason you went with _blank? As far as I can tell, this isn't consistent with the other links on the advanced search page either.

Reply:
Good point.  Our 'Internet Archive Page' links from the individual works' pages should probably be the model here.